src/components/DatePicker/DatePicker.tsx

```diff
@@ -88,16 +101,29 @@ export const DatePicker: FC<Props> = ({
     setIsCalendarShown(true)
     const rect = inputWrapperRef.current.getBoundingClientRect()
     setCalendarPosition({
-      top: rect.top + rect.height - 4,
-      left: rect.left,
+      top: rect.top + rect.height - 4 + window.pageYOffset,
+      left: rect.left + window.pageXOffset,
     })
   }, [])
 
   useEffect(() => {
-    if (date && inputRef.current) {
-      inputRef.current.value = dateToString(date)
+    if (!value || !inputRef.current) {
+      return
     }
-  }, [date, dateToString])
+    /**
+     * Do not format the given value in the following cases
+     * - while input element is forcused.
```

Comment:
```suggestion
     * - while input element is focused.
```

しょうもないコメントですが typo です！ 👮

Reply:
ありがとうございます！修正しました！